src/battle_script_commands.c

```diff
@@ -13144,12 +13144,22 @@ static void Cmd_trychoosesleeptalkmove(void)
     }
 }
 
-static void Cmd_setdestinybond(void)
+static void Cmd_trysetdestinybond(void)
 {
-    CMD_ARGS();
+    CMD_ARGS(const u8 *failInstr);
 
-    gBattleMons[gBattlerAttacker].status2 |= STATUS2_DESTINY_BOND;
-    gBattlescriptCurrInstr = cmd->nextInstr;
+    if (B_UPDATED_MOVE_DATA >= GEN_7 && gDisableStructs[gBattlerAttacker].usedDestinyBond)
+    {
+        gDisableStructs[gBattlerAttacker].usedDestinyBond = FALSE;
+        gBattlescriptCurrInstr = cmd->failInstr;
+    }
+    else
+    {
+        if (B_UPDATED_MOVE_DATA >= GEN_7)
```

Comment:
We usually do standalone configs for move effects.

Reply:
Oh sure! I don't often do these, fixed :)